playbook.yml

```diff
@@ -316,6 +317,11 @@
           description: "Jenkins HMCTS GitHub credentials (Username/password format)"
           username: "{{ api_key_username_ctsc }}"
           password: "{{ lookup('azure_keyvault', 'secret_name=hmcts-github-apikey-ctsc vault_uri={{ vault_uri }}', errors='warn') }}"
+        - id: jenkins-github-hmcts-api-token_ethos
```

Comment:
did you want a build monitor as well? if so modify cac-prod.yml

Reply:
Added buildMonitor as well